Source/ProgressionSystemRuntime/Public/Components/PSSpotComponent.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Data/PSTypes.h"
+#include "PSSpotComponent.generated.h"
+
+/**
+ * Represents a spot where a character can be selected in the Main Menu.
+ * Is added dynamically to the My Skeletal Mesh actors on the level.
+ */
+UCLASS(Blueprintable, BlueprintType, ClassGroup = (Custom), meta = (BlueprintSpawnableComponent))
+class PROGRESSIONSYSTEMRUNTIME_API UPSSpotComponent : public UActorComponent
```

Comment:
Please include the parent with  '#include "Components/ActorComponent.h"', otherwise it will fail the cook

Reply:
@h4rdmol ping

Reply:
done